### ISSUE TITLE
SyntaxError on `format()` call with newline before first arg

### ISSUE THREAD

**Issue by snarfed** [ikamensh/flynt#115]
Hi again! Here's another SyntaxError I noticed. Feel free let me know if these are expected failures, I won't file any more if so. 😁

`x.py`:
```py
var = {}
'foo {}'.format(
    var.get('bar'))
```

```sh
$ flynt -v ~/x.py
Running flynt v.0.69
Failed to parse transformed code 'f'foo {var.get('bar')}'' given original ''foo {}' . format ( var . get ( 'bar' ) )'
f-string: unmatched '(' (<unknown>, line 1)
Traceback (most recent call last):
  File "/Users/ryan/src/oauth-dropins/local/lib/python3.9/site-packages/flynt/transform/transform.py", line 49, in transform_chunk
    ast.parse(new_code)
  File "/opt/homebrew/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 1
    f'foo {var.get('bar')}'
...
```

It works fine if I move the arg up to the same line:

```py
var = {}
'foo {}'.format(var.get('bar'))
```

**Comment by ikamensh** [comment]
It seems that the problem is quote types in the transformed output `f'foo {var.get('bar')}'`.

**Comment by ikamensh** [comment]
Fixed in v0.73
